processor/src/operations/sys_ops/sys_event_handlers.rs

```diff
@@ -342,6 +346,45 @@ pub fn push_u64_div_result(
     Ok(())
 }
 
+/// Pushes the result of divison (both the quotient and the remainder) of a [u64] by the Falcon
+/// prime (M = 12289) onto the advice stack.
+///
+/// Inputs:
+///   Operand stack: [a1, a0, ...]
+///   Advice stack: [...]
+///
+/// Outputs:
+///   Operand stack: [a1, a0, ...]
+///   Advice stack: [q0, q1, r, ...]
```

Comment:
```suggestion
///   Advice stack: [q1, q0, r, ...]
```

Based on the implementation

Reply:
I think we want that `adv_push.1` pushes `q0` first and when we call again `adv_push.1` it pushes `q1`.

Reply:
Actually, what I wrote was wrong, we want the first `adv_push.1` to push `q1`, so your suggestion is correct.
Will fix it, thanks!